### DISCUSSION TITLE
posts/efcore-minimal-example/

### DISCUSSION THREAD

**Discussion by giscus** [BYJRK/byjrk.github.io#5] [Announcements]
# posts/efcore-minimal-example/

# 安装 EntityFrameworkCore 以 Sql Server 为例，可以在 NuGet 包管理器中搜索并安装以下包： Microsoft.EntityFrameworkCore Microsoft.EntityFrameworkCore.SqlServer # 实现 Model 假定现在有这样一张表： 1 2 3 4 5 6 CREATE TABLE [dbo].[Blog] ( [BlogId] INT IDENTITY (1, 1) NOT NULL, [Title] NVARCHAR (100) NULL, [Author] NVARCHAR (50) NULL,

https://blog.coldwind.top/posts/efcore-minimal-example/

**Comment by ZerosZhang** [comment]
我经常使用的是SqlSugar这个包，也挺好用

**Comment by Zhaozhuofan** [comment]
如果Blogging数据库中有很多表,EFCore是通过什么判断自己Blogs属性对应了数据库中的Blog表呢?

**Reply by BYJRK** [reply]
EF 默认有一套命名规范，包括单复数等，所以默认是根据名称来的。如果你希望定制名称，那么一般的做法是给类添加 `TableAttribute`，或者重写 `DbContext` 的 `OnModelCreating` 方法

**Reply by Zhaozhuofan** [reply]
谢谢大佬的回复!!!
顺便问一下,近期有做EFCore教学视频的打算嘛?

**Reply by BYJRK** [reply]
以后应该会有，但近期顾不上

**Reply by XVCoder** [reply]
近期打算用efcore8+MySQL开发一个工具，结果发现db-first那一套整不会了，上一次用还是efcore3 :cry:

**Reply by XVCoder** [reply]
说错了，是code-first

**Reply by BYJRK** [reply]
EF的code first还是挺好弄的，用它的CLI工具
https://learn.microsoft.com/en-us/ef/core/cli/dotnet

**Comment by rhodi2436** [comment]
ef 和 efcore 用哪个呢？我看官方推荐使用 ef

**Reply by rhodi2436** [reply]
是我记反了。🤡
